### ISSUE TITLE
reverse DNS resolving of hostnames in IPv6 mode results in IPv4 IPs

### ISSUE THREAD

**Issue by sunweaver** [cajus/uif#2]
I have something like the below in uif6.conf:

```
network {
     gdns google-public-dns-a.google.com.
}

filter {
    out+  p=dns s=gdns
}
```

When looking at it with `ip6tables -L`, I get the following:

```
Chain 8_1 (1 references)
target     prot opt source               destination         
ACCEPT     all      ::1/128              ::/0                
ACCEPT     all     google-public-dns-a.google.com /128   ::/0                
```

However, when I look at it with `ip6tables -L -n`, I get this:

```
Chain 8_1 (1 references)
target     prot opt source               destination         
ACCEPT     all      ::1/128              ::/0                
ACCEPT     all      ::8.8.8.8/128   ::/0                
```

My expectation would be:

```
Chain 8_1 (1 references)
target     prot opt source               destination         
ACCEPT     all      ::1/128              ::/0                
ACCEPT     all       2001:4860:4860::8888/128   ::/0
```

**Comment by sunweaver** [comment]
References:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=851388
https://rt.cpan.org/Ticket/Display.html?id=119858

**Comment by sunweaver** [comment]
Closing, as resolved by 91d3907a7ff141d87e681b5672b44bcbb1b17eae.
